generated/web/microsoft.web/2015-08-01/types.md

```diff
@@ -142,8 +179,8 @@
 * **id**: string (ReadOnly, DeployTimeConstant): The resource id
 * **kind**: string: Kind of resource
 * **location**: string (Required): Resource Location
-* **name**: 'discover' (Required, DeployTimeConstant): The resource name
-* **properties**: [RestoreRequestProperties](#restorerequestproperties)
+* **name**: 'discover' | string (Required, DeployTimeConstant): The resource name
+* **properties**: [RestoreRequestPropertiesOrBackupItemProperties](#restorerequestpropertiesorbackupitemproperties)
```

Comment:
This is another case where [the PUT side uses a constant string name](https://github.com/Azure/azure-rest-api-specs/blob/a0ff8cb6633156cddfe66e77944e3aba99a005be/specification/web/resource-manager/Microsoft.Web/stable/2015-08-01/service.json#L13441-L13500) and [the GET side uses a parameterized name](https://github.com/Azure/azure-rest-api-specs/blob/a0ff8cb6633156cddfe66e77944e3aba99a005be/specification/web/resource-manager/Microsoft.Web/stable/2015-08-01/service.json#L13671-L13776).

Looking at the semantics of the PUT method that's currently being captured as a resource, I believe this is an RP modeling error. The PUT method should be a POST action, and the `/backups` resource should be read only. That'd be a breaking change and would need to be fixed in the swagger.